### PR TITLE
adding new styles and illustrations for pos

### PR DESCRIPTION
**Pull request by sineade**
### PR Review Checklist
:art:
https://amaabca.visualstudio.com/driver_education_backlog/_workitems?id=2019&_a=edit
Not all the screens in pos were affected by these changes. Below are screenshots of what was affected.

screen shot 2017-09-13 at 2 26 51 pm
screen shot 2017-09-13 at 2 27 06 pm
screencapture-pos-dev-ama-ab-ca-customers-new-1505334436032

To be completed by the person who opens the PR. Use strikethroughs for items that are not applicable in list.
- [x] I have reviewed my own PR to check syntax & logic, removed unnecessary/old code, made sure everything aligns with our style guide and BEM.
- [x] I've added new components to the style guide (or have a PR in to add them).
- [x] Any applicable version numbers have been updated.
- [x] I've tested on mobile, tablet, and desktop, as well as across all of the browsers we support.
- [x] I've proof-read all text for legibility, proper grammar, punctuation, and capitalization (titlecase).
- [x] I have written a detailed PR message explaining what is being changed and why
- [x] I’ve linked to any relevant VSTS tickets
- [x] I’ve added the appropriate review symbols to my PR - (elephant) for dev review, (art) for design

### CODE REVIEW

File: app/assets/stylesheets/layout/helper-classes.scss [hunk 1]
```diff
@@ -269,6 +269,19 @@
   }
 }
 
+.border-right{
+  border-right: 2px solid $stone;
+  @include breakpoint(small down){
```
Comment (DaveAMA) [inline app/assets/stylesheets/layout/helper-classes.scss]: space between media query and `border-right`

**Comment by DaveAMA** [comment]
💩 Just one small change

**Comment by DaveAMA** [comment]
forgot to add :book:

**Comment by sineade** [comment]
:wave:

File: app/assets/stylesheets/layout_components/tabs.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+.tabs-title.is-active{
+  background-color: $cerulean;
+  a{
+    color: $white;
+    background-color: $cerulean;
+  }
+  a:hover{
+    background-color: $cerulean;
+  }
+}
```
Comment (DaveAMA) [inline app/assets/stylesheets/layout_components/tabs.scss]: spacing in this file

**Comment by DaveAMA** [comment]
💩

**Comment by sineade** [comment]
:wave:

**Comment by DaveAMA** [comment]
🎨 🎨 🎨

**Pull request merged**